kusto/kusto_examples_test.go

```diff
@@ -138,39 +135,31 @@ func Example_complex() {
 }
 
 func ExampleAuthorization_config() {
```

Comment:
Names have to be logical as per go standards , for a function called config it should be TestConfig()